### DISCUSSION TITLE
I encountered new problems in the new construction

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#72] [Q&A]
If this question volume recovery parameter, the link is as follows:http://www.abcdiaocha.com/joggle.html?order=662262&kds=c@yxB0fOwhFM4Xmk3806m39W964Wtm03xM&abcid=e1777bc40597c1ce
After completing and successful identification, Submit has been clicked; the rewarding point page is not rewarded, but in fact the homepage data has been updated; the newly generated answer points have been added.
I used Chromium GitHub to encapsulate the version of the code library and tried another questionnaire link. This questionnaire was not the same level, and the screening did not belong to the target group; The main station is: http://www.abcdiaocha.com needs to register the investigation member account in order to answer the questions and analyze the problem; thank you for your cooperation!

**Comment by gz83** [comment]
> If this question volume recovery parameter, the link is as follows:http://www.abcdiaocha.com/joggle.html?order=662262&kds=c@yxB0fOwhFM4Xmk3806m39W964Wtm03xM&abcid=e1777bc40597c1ce After completing and successful identification, Submit has been clicked; the rewarding point page is not rewarded, but in fact the homepage data has been updated; the newly generated answer points have been added. I used Chromium GitHub to encapsulate the version of the code library and tried another questionnaire link. This questionnaire was not the same level, and the screening did not belong to the target group; The main station is: http://www.abcdiaocha.com needs to register the investigation member account in order to answer the questions and analyze the problem; thank you for your cooperation!

@whxuezhe Does this problem exist in other browsers?

**Reply by whxuezhe** [reply]
@gz83 Great, you don't seem to look at my description seriously; Chromium Github encapsulates the version of the code library. When the parameter is Screenout, the callback is successful and the questionnaire has not successfully participated in the reward.
Roughly, other browsers based on Chromium's cores are mostly implemented by the callback parameters. In our domestic answering discussion group, this is the case; please know.

**Reply by gz83** [reply]
> @gz83 Great, you don't seem to look at my description seriously; Chromium Github encapsulates the version of the code library. When the parameter is Screenout, the callback is successful and the questionnaire has not successfully participated in the reward.

@whxuezhe I saw that in your description, there may be unexpected behavior on the webpage, so I sent the previous reply.  So now, do you think this is a browser issue or a website issue?

**Reply by whxuezhe** [reply]
@gz83 If the judgment is the problem of the webpage itself, it should be the related JS code framework of the webpage and the questionnaire system, which is not suitable for this browser; if it is a browser, it is the reason for the browser V8 interpreter. When calling the message, there are potential problems on the JS code instruction.

**Reply by gz83** [reply]
> @gz83 If the judgment is the problem of the webpage itself, it should be the related JS code framework of the webpage and the questionnaire system, which is not suitable for this browser; if it is a browser, it is the reason for the browser V8 interpreter. When calling the message, there are potential problems on the JS code instruction.

@whxuezhe So I have already asked you whether other browsers have related problems, but you did not reply, which already shows that you did not cooperate with the investigation.

**Reply by whxuezhe** [reply]
@gz83 It is known that the 360 safety browser core 108 and 360 speed browser core 95 can be performed normally. Chromium GitHub encapsulates the native Chromium browser of the code library. I have told you twice that you can execute the Screenout, that is, the scoring instruction parameters; the completion and success should not have problems, and it is difficult to execute.

**Reply by gz83** [reply]
> @gz83 It is known that the 360 safety browser core 108 and 360 speed browser core 95 can be performed normally. Chromium GitHub encapsulates the native Chromium browser of the code library. I have told you twice that you can execute the Screenout, that is, the scoring instruction parameters; the completion and success should not have problems, and it is difficult to execute.

@whxuezhe Try it with the latest AVX version of Thorium, don't use the AVX2 version for now, and see if you can reproduce the problem you encounter

In addition, I would like to say that some content in your previous description does not conform to the logic of expression, please pay attention to the expression of content.
